### DISCUSSION TITLE
Would like a GO Midi Recording

### DISCUSSION THREAD

**Discussion by 5763-6146** [GrandOrgue/grandorgue#755] [Help]
I would like to find a Midi file that I can use to demonstrate my recent GO conversion. I have recorded myself playing something trivial, but it would be nice to have a few minutes or more of someone competent playing. Is it possible to get such a file? I have the Go sample organ loaded, but could load one of the others. Thanks

**Comment by midimusic** [comment]
Have a look at https://midimusic.github.io there are many hundreds of tunes you can play on organs. All of them can be translated (using eplayOrgan) to play on the GO demo organ or any other GO organ. Not only that eplayOrgan can both record and play midi files in GO format.

You will learn a lot by making use of this music.

csw900

**Reply by 5763-6146** [reply]
Dear csw900: I have downloaded eplayOrgan and read the very nice instructions. Very impressive! I also got a Wurlitzer midi file (of a song I like) to play on eplay. So far, so good.
Question: all the music on the site you suggest seems to be popular music for a theater organ, not classical. 
Where would I find a classical midi file for a church organ? It could be short e.g. some of Lars' short demos that he does on his Youtube instructional videos, or anything else hopefully recognizable,   Thanks

**Reply by 5763-6146** [reply]
I found some midi files of classical music that play in eplayOrgan. The rest is up to me, following the instructions. Thanks!

**Comment by midimusic** [comment]
Here is a "classical" midi file for the GO demo organ to get you going. It is Mozart K525 you should recognise it when you play it.

It was recorded on a newly downloaded GO v3.2.0-1. I was disappointed to find that the volume it still set too low at -15 and that the fake reverb was set at maximum. Much better default settings would be -6 and 2000ms.

[mozartK525_GOdemo.zip](https://github.com/GrandOrgue/grandorgue/files/7311539/mozartK525_GOdemo.zip)

csw900

**Reply by 5763-6146** [reply]
Wonderful!!! It is just what I was looking for. Thank you very much.
